### DISCUSSION TITLE
Native JSON data type support

### DISCUSSION THREAD

**Discussion by JichaoS** [lquerel/gcp-bigquery-client#51] [Ideas]
Hello!

BigQuery supports the JSON data type: https://cloud.google.com/bigquery/docs/reference/standard-sql/json-data

Are there any plans to support this datatype in querying?

**Comment by lquerel** [comment]
This is definitely something we need to add in this crate. However, I don't know yet when I will be able to add it, as my current schedule is already quite busy. 

Note that this crate is open for contribution.
